logprep/util/getter.py

```diff
@@ -56,11 +69,11 @@ def from_string(cls, getter_string: str) -> "Getter":
         if protocol is None:
             protocol = "file"
         if protocol == "file":
-            return FileGetter(protocol=protocol, target=target)
+            return FileGetter(protocol=protocol, target=target)  # type: ignore
         if protocol == "http":
-            return HttpGetter(protocol=protocol, target=target)
+            return HttpGetter(protocol=protocol, target=target)  # type: ignore
         if protocol == "https":
-            return HttpGetter(protocol=protocol, target=target)
+            return HttpGetter(protocol=protocol, target=target)  # type: ignore
```

Comment:
Do we still need those ignores after fixing the problem with `_shared`?

Reply:
I was hoping we wouldn't , but mypy still complains, even now.